vaadin-button.html

```diff
@@ -22,25 +22,178 @@
   <template>
     <style>
       :host {
-        display: block;
+        align-items: flex-start;
+        background: linear-gradient(to bottom right, #fff, #eee);
+        border: 1px solid #999;
+        border-radius: 2px;
+        box-sizing: border-box;
+        cursor: default;
+        display: inline-block;
+        font-size: 15px;
+        letter-spacing: normal;
+        margin: 0;
+        padding: 2px 8px;
+        text-align: center;
+        text-indent: 0;
+        text-rendering: auto;
+        text-shadow: none;
+        text-transform: none;
+        word-spacing: normal;
+        user-select: none;
+      }
+
+      :host(:hover) {
+        border-color: #666;
+      }
+
+      :host([disabled]) {
+        color: #aaa !important;
+        background: #eee !important;
+        border-color: #aaa !important;
+        pointer-events: none;
+      }
+
+      :host([active]) {
+        background: linear-gradient(to bottom right, #eee, #fff);
       }
     </style>
+    <slot id="label"></slot>
   </template>
 
   <script>
-    class VaadinElement extends Polymer.Element {
-      static get is() {
-        return 'vaadin-button';
-      }
+    HTMLImports.whenReady(function() {
```

Comment:
Assuming that this file is imported by HTML Improts as well, this is unnecessary. HTML Imports implementation should handle this timing right.